tests/io/test_parquet.py

```diff
@@ -130,3 +132,20 @@ def test_parquer_write(dataset, tmp_path):
     pf = pq.ParquetFile(tmp_path / "foo.parquet")
     output_table = pf.read()
     assert dataset.data.table == output_table
+
+
+@require_pil
+def test_dataset_to_parquet_keeps_features(shared_datadir, tmp_path):
+    import PIL.Image
+
+    image_path = str(shared_datadir / "test_image_rgb.jpg")
+    PIL.Image.fromarray(np.zeros((5, 5), dtype=np.uint8)).save(image_path, format="png")
```

Comment:
Oh actually you can also remove this line, the file already exists and is a dummy image we use for tests

Reply:
Done! The test actually does not use `PIL` anymore after that so I also dropped the `require_pil`.

Reply:
`require_pil` is actually required to use image datasets, so you can re-add it ^^

Reply:
re-added ^^ sorry about that